### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by a-velasco**
## Issue
README.md is out of sync with latest updates to the [MongoDB home page on charmhub.io](https://charmhub.io/mongodb)
Jira issue: [DPE-3747](https://warthogs.atlassian.net/browse/DPE-3747)

## Solution
- Edited the text to match home page more closely
- Removed unnecessary or repeated information
- Added further links and resources 

[DPE-3747]: https://warthogs.atlassian.net/browse/DPE-3747?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Mehdi-Bendriss — DISMISSED** [review]

**Review by MiaAltieri — DISMISSED** [review]
this looks excellent and your work is greatly appreciated!

 Be sure Michelle reviews this - she was the one who wrote this initially.

Also I believe the text here should match the overview page on the charmhub docs

File: README.md [hunk 1]
```diff
@@ -5,59 +5,46 @@
...
 
-Canonical has developed an open-source operator called  Charmed MongoDB, making it easier to operate MongoDB. The Charmed MongoDB Virtual Machine (VM) operator deploys and operates MongoDB on physical,  Virtual Machines (VM) and other wide range of cloud and cloud-like environments, including AWS, Azure, OpenStack and VMWare.
-
-Charmed MongoDB (VM Operator) is an enhanced, open source and fully-compatible drop-in replacement for the MongoDB Community Edition with advanced MongoDB enterprise features. It simplifies the deployment, scaling, design and management of MongoDB in production in a reliable way. In addition, you can use the operator to manage your MongoDB clusters with automation capabilities. It also offers replication, TLS, password rotation, easy-to-use application integration, backup, restore, and monitoring.  
-
-Aside from a VM operator, Canonical also developed another operator called [Charmed MongoDB (K8s operator)](https://charmhub.io/mongodb-k8s?channel=5/edge) that operates in the Kubernetes environment, and you can see the documentation [here](https://charmhub.io/mongodb-k8s?channel=5/edge).
+It currently supports:
+* MongoDB deployment
```
Comment (MiaAltieri) [inline README.md]: I would remove and/or reword

File: README.md [hunk 2]
```diff
@@ -5,59 +5,46 @@
...
+* Replication
+* Sharding
+* Password set and rotation
```
Comment (MiaAltieri) [inline README.md]:
```suggestion
* Password rotation
```

File: README.md [hunk 3]
```diff
@@ -5,59 +5,46 @@
...
+* User management
+* Client connections via integrations
```
Comment (MiaAltieri) [inline README.md]:
```suggestion
* Client connections
```

File: README.md [hunk 4]
```diff
@@ -5,59 +5,46 @@
...
+* TLS encryption
+* Backup & restore
+* Monitoring
+* Auditing
```
Comment (MiaAltieri) [inline README.md]:
```suggestion
* Audit logging
```

File: README.md [hunk 5]
```diff
@@ -5,59 +5,46 @@
...
+* Observability
```
Comment (MiaAltieri) [inline README.md]:
```suggestion
```
remove this - same thing as monitoring

**Review by Mehdi-Bendriss — APPROVED** [review]

**Review by MiaAltieri — APPROVED** [review]

**Pull request merged**